README.md

```diff
@@ -8,10 +8,13 @@
 
 ## Posting
 * Fork this repo
-* Locally, and after installation, make a new file in `_posts/`
-  * Title the file `YYYY-MM-DD-your-title-here.md` (i.e. `2017-10-23-hello-world.md`)
-  * Fill in the YAML Front Matter at the top of the file, as detailed [below](#post-format)
-  * Below the Front Matter, write the body of the post, in Markdown
+* Locally, and after installation, either:
+  * Run `sh newPost.sh` and fill in metadata to automatically generate a new post file
+    * Fill in the post content, and verify YAML Front Matter **OR:**
+  * Make a new file in `_posts/`:
```

Comment:
Is this an alternative step to running the aforementioned script? It reads as if this needs to be done after running the script and I don't believe that is the case.

Reply:
Yes, the person *either* 
1) Runs `newPost.sh`
**or**
2) Fills in the YAML themselves

I bolded the "**OR:**", so I'm not sure how I can be more clear on that.

Reply:
I totally didn't read the "**OR:**" so maybe that would be better on it's own bulletpoint? That's more of a nitpick though.